Sources/Orbit/Support/Layout/FrameWidthAtLeastHeightModifier.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+
+struct FrameWidthAtLeastHeightModifier: ViewModifier {
+
+    @State var height: CGFloat = 0
+
+    public func body(content: Content) -> some View {
+        ZStack {
+            Color.clear
+                .frame(width: height, height: 1)
+
+            ContentHeightReader(height: $height) {
+                content
+            }
+        }
+    }
+}
+
+extension View {
+
+    /// Ensures the width to be at least the same size as height.
+    func frameWidthAtLeastHeight() -> some View {
```

Comment:
To me, it looks like that width can't be bigger than height, so maybe we can rename to `frameWidthAsHeight`?

Reply:
That would mean it is exactly as height, thats why I put `AtLeast` there - but I am open to other suggestions, the name is horrible 😄